### ISSUE TITLE
"ARTShape" was not found in the UIManager

### ISSUE THREAD

**Issue by KalidassM01** [mohebifar/react-native-loader#20]
Hi,

![Screen Shot 2020-04-19 at 12 47 41 PM](https://user-images.githubusercontent.com/49358979/79681914-d5cec280-823b-11ea-96f8-5cd828ac4e03.png)

Im using React native Expo(36.0.0). 
I am developing dashboard screen for my project. So, I just want to show some pie charts in dashboard screen. I have tried many libraries for all kind of charts. Nothing worked for me. Finally I have used **react-native-pie** with the help of youtube video. Still it is showing me a same error.

 I am new to React native.

Thanks!

**Comment by piyush-ramavat** [comment]
ART has been removed from react-native SDK v 36 onwards.

Developers of this library will have to remove import ART from react-native and add directly the community library..

Code:

```
import {ART} from 'react-native';
const {Surface, Shape} = ART;
```

to:

`import {Surface, Shape} from '@react-native-community/art';`

**Comment by piyush-ramavat** [comment]
There is a PR already pending with the required change.
@mohebifar could you please review - merge this PR asap?
Thanks

https://github.com/mohebifar/react-native-loader/pull/17/files

**Comment by mohebifar** [comment]
Published in v1.3.1. Thanks!

**Comment by Lokeshc123** [comment]
still artshape not found using the new package
